<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.269">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>EPI519 Evolutionary Epidemiology - 2&nbsp; Week 2</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../week1/week1.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>

  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Week 2</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">EPI519 Evolutionary Epidemiology</a> 
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../week1/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Week 1</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../week2/week2.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Week 2</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#diversity" id="toc-diversity" class="nav-link active" data-scroll-target="#diversity"><span class="toc-section-number">3</span>  Diversity</a>
  <ul class="collapse">
  <li><a href="#bacteria" id="toc-bacteria" class="nav-link" data-scroll-target="#bacteria"><span class="toc-section-number">3.1</span>  Bacteria</a></li>
  <li><a href="#dna-transfer-recombination" id="toc-dna-transfer-recombination" class="nav-link" data-scroll-target="#dna-transfer-recombination"><span class="toc-section-number">3.2</span>  DNA transfer, recombination</a>
  <ul class="collapse">
  <li><a href="#summary-of-bacteria" id="toc-summary-of-bacteria" class="nav-link" data-scroll-target="#summary-of-bacteria"><span class="toc-section-number">3.2.1</span>  Summary of Bacteria</a></li>
  </ul></li>
  <li><a href="#viruses" id="toc-viruses" class="nav-link" data-scroll-target="#viruses"><span class="toc-section-number">3.3</span>  Viruses</a></li>
  <li><a href="#eukaryotic" id="toc-eukaryotic" class="nav-link" data-scroll-target="#eukaryotic"><span class="toc-section-number">3.4</span>  Eukaryotic</a></li>
  <li><a href="#fungi" id="toc-fungi" class="nav-link" data-scroll-target="#fungi"><span class="toc-section-number">3.5</span>  Fungi</a></li>
  <li><a href="#strategies-for-dealing-with-immunity" id="toc-strategies-for-dealing-with-immunity" class="nav-link" data-scroll-target="#strategies-for-dealing-with-immunity"><span class="toc-section-number">3.6</span>  Strategies for dealing with immunity</a></li>
  <li><a href="#summary" id="toc-summary" class="nav-link" data-scroll-target="#summary"><span class="toc-section-number">3.7</span>  Summary</a></li>
  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="toc-section-number">3.8</span>  Discussion</a></li>
  </ul></li>
  <li><a href="#transmission-routes-adaptation-and-emergence" id="toc-transmission-routes-adaptation-and-emergence" class="nav-link" data-scroll-target="#transmission-routes-adaptation-and-emergence"><span class="toc-section-number">4</span>  Transmission Routes, Adaptation, and Emergence</a>
  <ul class="collapse">
  <li><a href="#zoonoses" id="toc-zoonoses" class="nav-link" data-scroll-target="#zoonoses"><span class="toc-section-number">4.1</span>  Zoonoses</a></li>
  <li><a href="#population-size" id="toc-population-size" class="nav-link" data-scroll-target="#population-size"><span class="toc-section-number">4.2</span>  Population Size</a>
  <ul class="collapse">
  <li><a href="#rough-n_es-for-some-famous-things" id="toc-rough-n_es-for-some-famous-things" class="nav-link" data-scroll-target="#rough-n_es-for-some-famous-things"><span class="toc-section-number">4.2.1</span>  Rough <span class="math inline">\(N_e\)</span>s for some famous things:</a></li>
  </ul></li>
  <li><a href="#bottlenecks-and-host-jumps" id="toc-bottlenecks-and-host-jumps" class="nav-link" data-scroll-target="#bottlenecks-and-host-jumps"><span class="toc-section-number">4.3</span>  Bottlenecks and Host Jumps</a></li>
  <li><a href="#review" id="toc-review" class="nav-link" data-scroll-target="#review"><span class="toc-section-number">4.4</span>  Review</a></li>
  <li><a href="#one-dose-or-two" id="toc-one-dose-or-two" class="nav-link" data-scroll-target="#one-dose-or-two"><span class="toc-section-number">4.5</span>  One Dose or Two</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Week 2</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="diversity" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Diversity</h1>
<p>The state of COVID-19 in this time last year (2022):</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/covid_in_2022.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">trees of COVID strains</figcaption><p></p>
</figure>
</div>
<p>Name some Is plasmodium more related to humans or bacteria? Humans! Because it’s eukaryotic.</p>
<ul>
<li>Clostridium — a bacteria.</li>
<li>Yersinia (including pseudo-tuberculosis, black death, etc.)</li>
<li>Candida aurus<br>
</li>
<li>Prions — infectious by virtue of their shape
<ul>
<li>Cause of bovine encephalitis, kuru, etc</li>
</ul></li>
<li>Amoeba</li>
<li>Ascaris worm</li>
<li>Baltic fishtape worm gets to 30 feet in length</li>
</ul>
<p>Basically everything except archaea have been found to be infectious.</p>
<p>Getting the biology right matters. Not only to avoid needless antibiotic prescriptions. Bacteria, viruses, protozoa are very different!</p>
<p><img src="images/batman.jpg" class="img-fluid"></p>
<p><img src="images/tree_of_life.png" class="img-fluid"></p>
<p>Everything that can be seen with the naked eye is contained in the three nodes labeled: homo, coprius (fungi), zea (maize).</p>
<p>Why are archaea not pathogenic?</p>
<blockquote class="blockquote">
<p>Gill, E., &amp; Brinkman, F. (2011). The proportional lack of archaeal pathogens: Do viruses/phages hold the key? BioEssays, 33 (4), 248-254 DOI: 10.1002/bies.201000091</p>
</blockquote>
<p>(Bill doesn’t necessarily agree with the think-piece above, just included to show you can write such a think-piece).</p>
<section id="bacteria" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="bacteria"><span class="header-section-number">3.1</span> Bacteria</h2>
<ul>
<li>Unicellular*</li>
<li>Circular genome, no nucleus*</li>
<li>Bacteria multiply by binary fission</li>
<li>Extraordinary diverse variation in gene content allows flexibility</li>
</ul>
<p>* The exceptions to unicellularity is that many can grow in chains or complex biofilms with properties linked to virulence. Also, some linear genomes known. Also contain plasmids</p>
<p>Borellia burgdorferi has linear chromosomes.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/genome_size.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">size of genomes of different pathogenic agents</figcaption><p></p>
</figure>
</div>
<p>In 2002, <a href="https://www.pnas.org/doi/abs/10.1073/pnas.252529799">Welch et al</a> showed that three strains of <em>E coli</em> only had 39% of genes shared. This variation explain why some strains cause diarrhea vs.&nbsp;other symptoms.</p>
<p><a href="https://c2-d2.github.io/hanage-lab/The-Amazing-Accessory-Genome-in-S.pneumoniae/" class="uri">https://c2-d2.github.io/hanage-lab/The-Amazing-Accessory-Genome-in-S.pneumoniae/</a></p>
<p>In general, free-living organisms have larger genomes because they experience a wider variety of conditions.</p>
<p>Pathogens, on the other hand, which only rely on a single species, tend to have smaller genomes because they’ve often outsourced all possible functionalities.</p>
<p>Obligate endosymbionts (chlamydoffila/chlamydia) have the smallest genomes of all. Often, one can see that their genes have even been copied into the eukaryotic host.</p>
</section>
<section id="dna-transfer-recombination" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="dna-transfer-recombination"><span class="header-section-number">3.2</span> DNA transfer, recombination</h2>
<p>DNA may be transmitted between lineages in bacteria via conjugation, transduction, transformation.</p>
<p><img src="images/gene_transfer.png" class="img-fluid"></p>
<ul>
<li>Conjugation: Picking up a plasmid</li>
<li>Transduction: Transfer via viral delivery</li>
<li>Transformation: sucking up DNA from the environment</li>
</ul>
<p>Why suck up DNA from the environment?</p>
<p>Making DNA is quite energy costly, so sucking up DNA can be food.</p>
<ul>
<li>Genome hypermobility by lateral transduction, Science 2018</li>
<li>Evolution via recombination: Cell-to-cell contact PLoS Genetics</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/mosaic.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Recombination between different Neisseria species</figcaption><p></p>
</figure>
</div>
<p>These DNA codes can contain features for antibiotic resistance.</p>
<p>People use the word ‘clonal’ to describe how much recombination happens.</p>
<p><img src="images/clonality.png" class="img-fluid"></p>
<p>The result is that we can’t resolve deep evolutionary relationships. This raises questions about what ‘species’ are.</p>
<p>Example: When studying a lineage in about 2014, about 40% of its genome had been replaced by recombination.</p>
<p>Instead of a tree of life, we have a temporal network.</p>
<p>Jim Mallet with David Reich showed that butterfly species in the Andes have exchanged gene sequences that determine the colors and patterns on their wings because one of them tastes bad and one of them doesn’t. (Mimicry).</p>
<section id="summary-of-bacteria" class="level3" data-number="3.2.1">
<h3 data-number="3.2.1" class="anchored" data-anchor-id="summary-of-bacteria"><span class="header-section-number">3.2.1</span> Summary of Bacteria</h3>
<ul>
<li>Most bacteria that do come into contact with humans only act as ‘accidental’ pathogens (0.36% of known can cause disease).</li>
<li>Very few are obligate, human-specific</li>
<li>Inverse relationship between niche specialization and genome size</li>
<li>Mr.&nbsp;Potato head</li>
<li>Small genome species: highly specialized for existence within a particular host niche</li>
</ul>
</section>
</section>
<section id="viruses" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="viruses"><span class="header-section-number">3.3</span> Viruses</h2>
<p>Since 2013 the International Committee on the Taxonomy of Viruses has said:</p>
<ul>
<li>““A [viral] species is a&nbsp;monophyletic&nbsp;group of viruses whose properties can be distinguished from those of other species by multiple criteria”</li>
<li>(like capsid structure, host range, enveloped or no among other things)</li>
</ul>
<p><img src="images/phylogeny_principles.png" class="img-fluid"></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/hiv_genes.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">implication: HIV can’t be viewed as a virus</figcaption><p></p>
</figure>
</div>
<p>Viruses are 20-300 nm in diameter.</p>
<blockquote class="blockquote">
<p>“For a dead thing, they make a lot of trouble” —Appropriately said on Halloween</p>
</blockquote>
<p>They get others to do things for them, much like PIs or journals.</p>
<p>Most of the estimated <span class="math inline">\(10^{31}\)</span> viruses on Earth are phages that infect bacteria (they kill between 4–50% of the bacteria produced every day in the environment)</p>
<p>Bill says that the Lancet published a letter claiming SARS came from space. He and others wrote eagerly awaiting for hard evidence of its implications: that there is eukaryotic life in space capable of sustaining viral evolution.</p>
<p>Viruses are:</p>
<ul>
<li>metabolically inert,</li>
<li>replicate only within the cells of living hosts</li>
<li>Have a RNA or DNA genome, a protein coat, and, in more complex types, a surrounding envelope.</li>
</ul>
<p>RNA vs.&nbsp;DNA: * Both nucleic acids – ribonucleic and deoxyribonucleic * We use double stranded DNA, the double helix * But viruses are much more variable</p>
<p>Viruses typically have high mutation rates — especially RNA viruses</p>
<ul>
<li>RNA is less stable than DNA</li>
<li>A single strand is less stable than a double strand</li>
<li>The things that copy RNA can’t correct their mistakes</li>
<li>Retroviruses use RNA, but copy it to DNA (using reverse transcriptase)</li>
<li>All these things mean that viruses generate variation quickly</li>
</ul>
<p><img src="images/correlation_w_genome_size.png" class="img-fluid" alt="Sanjuan et al (2010) J Virol"> <img src="images/genome_size.png" class="img-fluid"></p>
<p>Each genome can only accumulate a limited amount of change per generation, thought to be due to not-coincidental. If something mutates too slowly, there’s not sufficient variation for adaptation and evolution to occur. If there’s too much, then there’s potential for a mutational meltdown.</p>
<p><img src="images/mutational_meltdown.png" class="img-fluid"></p>
<p><img src="images/flu_error.png" class="img-fluid"></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/recombination_in_viruses.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Recombination in viruses – it’s not just for bacteria</figcaption><p></p>
</figure>
</div>
<p>Because influenza has 9 “chunks of genes” (chromosomes?) that are already segmented — so you don’t need anything special for recombination. If one cell gets infected by multiple flu viruses, the segments can recombin. Each segment codes for different proteins that are part of the virus.</p>
<p>We call this <em>antigenic shift</em>.</p>
<p><img src="images/antigenic_shift.png" class="img-fluid"></p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/h1n1.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">H1N1 from 2009</figcaption><p></p>
</figure>
</div>
<p>How can we be sure something is a swine virus? We can see that it’s picked up genetic material from other swine viruses.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/betacoronavirus.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">SARS-CoV-2 is a betacoronavirus; closest relatives are bat coronaviruses separated by decades</figcaption><p></p>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/slow_evolution_virus.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">A ~30kb positive sense single stranded RNA virus has very slow mutation</figcaption><p></p>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/virus_error_correction.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">In fact, coronaviruses have their own</figcaption><p></p>
</figure>
</div>
<p>Recall, Bill’s definition: Change in allele frequency.</p>
<p>First published set of sequences from an immunocompromised host.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/immunocompromised.jpg" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Choi et al 2020, evolution in an immunocompromised host</figcaption><p></p>
</figure>
</div>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/alpha_mutations.png" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">B.1.1.7 or Alpha had more mutations than ‘expected’</figcaption><p></p>
</figure>
</div>
<p><img src="images/malnupiravir.png" class="img-fluid"></p>
</section>
<section id="eukaryotic" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="eukaryotic"><span class="header-section-number">3.4</span> Eukaryotic</h2>
<p>Like us in that their DNA is packaged with proteins in chromosomes And they can reproduce sexually, not that they always do.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/prokaryotic_vs_eukaryotic.jpg" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Eukaryotes are bigger than bacteria</figcaption><p></p>
</figure>
</div>
<p><img src="images/eukaryotic_pathogens.jpg" class="img-fluid"></p>
<p>Fungi, Algae, Protozoa, and Helminthes. Eukaryotic pathogen life forms are clearly dominated by the fungi and protists. Protist species are responsible for globally important diseases such as the malaria-causing Plasmodium species and the Leishmania and Trypanosoma species that cause significant mortality in the developing world.</p>
<p>Among the 20 most frequent microbial causes of death worldwide, six are eukaryotic.</p>
<p>“Two fold cost of sex” — many theories, Bill’s favorite being that it’s an adaptation to mitigate pathogens.</p>
<p><img src="images/cdc_plasmodium.png" class="img-fluid"></p>
<p>Typically evolution goes:</p>
<p>Mutation -&gt; Variation -&gt; Selective Pressure -&gt; Adaptation</p>
<p>Humans generate diversity in our immune function and response through combinations of diverse loci in the Major Histocompatibility Complex.</p>
<p>Similar strategies are followed by some parasites.</p>
<p><img src="images/var_genes.png" class="img-fluid"></p>
<p>Plasmodium have <em>var</em> genes. These are involved in attachment to red blood cells. About ~60 of them. One is expressed at a time, and exactly which varies during infection. Further variation is generated via recombination.</p>
<p>Trypanosomes — variable surface glycoproteins</p>
<ul>
<li>These are the cause of sleeping sickness and Chagas disease.</li>
<li>Trypanosomes are coated in a highly variable glycoprotein.</li>
</ul>
<p>There are &gt;1000 <em>vsg</em> genes in each parasite (!)</p>
</section>
<section id="fungi" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="fungi"><span class="header-section-number">3.5</span> Fungi</h2>
<p>Yeast infection (candidiasis) is caused by the yeast Candida albicans.</p>
<p><em>Apergillus</em> found in individuals with lung disease or cancer that have been exposed to fungal spores.</p>
<p>Candida auris is a fungal pathogen, with antimicrobial resistance, which is increasingly important.</p>
<p><img src="images/yeast_strains_bioRxiv.png" class="img-fluid"></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2023.10.17.562616v1" class="uri">https://www.biorxiv.org/content/10.1101/2023.10.17.562616v1</a></p>
</section>
<section id="strategies-for-dealing-with-immunity" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="strategies-for-dealing-with-immunity"><span class="header-section-number">3.6</span> Strategies for dealing with immunity</h2>
<ul>
<li>Rapid mutation – RNA viruses</li>
<li>Antigenic variation – parasites and bacteria</li>
<li>Recombination – used throughout life (and viruses to boot)</li>
</ul>
<p>For example, Neisseria meningitis has maintained less antigenic shift than Neisseria gonoccocus because droplet transmission can happen relatively quickly. On the other hand, gonorrhea has to stick around until it’s transmitted.</p>
</section>
<section id="summary" class="level2" data-number="3.7">
<h2 data-number="3.7" class="anchored" data-anchor-id="summary"><span class="header-section-number">3.7</span> Summary</h2>
<ul>
<li>With the exception of Archaea* all parts of the tree of life include pathogens</li>
<li>Bacteria – diverse, Mr potato heads</li>
<li>Viruses – slimmed down, can have high mutation rates</li>
<li>Eukaryotes – complex life cycles, capable of sexual reproduction.</li>
</ul>
<p>*Although archaea are part of the microbiota.</p>
</section>
<section id="discussion" class="level2" data-number="3.8">
<h2 data-number="3.8" class="anchored" data-anchor-id="discussion"><span class="header-section-number">3.8</span> Discussion</h2>
<ul>
<li>Why haven’t we identified the host of ebola?
<ul>
<li>Well, where do viruses get sampled? Where there are labs. Where there are resources. Where there are humans.</li>
</ul></li>
<li>What do you think of the Global Virome Project? Is it a good use of resoures? <a href="https://www.globalviromeproject.org/" class="uri">https://www.globalviromeproject.org/</a>
<ul>
<li>It’s useful for helping to elucidate the variation in viruses</li>
<li>Shining the flashlight into the darkness</li>
</ul></li>
</ul>
</section>
</section>
<section id="transmission-routes-adaptation-and-emergence" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Transmission Routes, Adaptation, and Emergence</h1>
<p>If a person with anthrax boarded a transatlantic flight, how many of their fellow passengers would you expected to be infected on arrival?</p>
<p>Anthrax is a bacillus, ancillus anthrasis, which produces a toxin.</p>
<p>The answer is nil because it does not transmit via humans — it transmits via spores.</p>
<p>Even the most deadly form of anthrax, which people have spent years trying to make transmissible, isn’t transmissible.</p>
<blockquote class="blockquote">
<p>What is adaptation? A fitness advantage playing off the existing variation. When existing variation is selected and the resulting is more likely to survive, that thing that emerges and that process are adaptation — both of them.</p>
</blockquote>
<p>How do pathogens adapt?</p>
<ul>
<li>What are they adapting to?</li>
<li>What are obstacles to adaptation?</li>
<li>How do they overcome these obstacles (if they do)?</li>
</ul>
<p>What do you think is being selected at different points?</p>
<p><img src="images/wastewater.png" class="img-fluid"> <img src="images/case_curve.png" class="img-fluid"></p>
<p>Simultaneously:</p>
<ul>
<li>Novelty in the spike protein, and</li>
<li>also achieving the functionality of infections</li>
</ul>
<p>D614G is an infection point developed very early — famous for increasing the affinity of the spike to the receptor. Alpha and delta also had similar mutations. More and more people developed immunity to the original virus. <em>Immune evasion</em> and <em>immune escape</em> are key. After that, the peak was really slow so it built up a lot of immunity. This poses a big survival pressure.</p>
<p>Do we think that if omicron infected such a large proportion, would this drive more variation or diversity in subsequent lineages? Actually, we’ve mostly seen omicron sub-lineages recently rather than distant lineages.</p>
<p>Early stage adaptation is all about host infectivity, and later on is all about immune evasion.</p>
<p>Shout-out some things that are being adapted:</p>
<ul>
<li>Infectiousness
<ul>
<li>The virus’s ability to infect or establish in a person</li>
<li>Reproductive number</li>
<li>Affinity of the receptor, receptor-binding</li>
</ul></li>
<li>Immune Evasion
<ul>
<li>From vaccine and infection acquired immunity (Convalescent, vaccine induced)
<ul>
<li>From B-cells, from T-cells</li>
</ul></li>
</ul></li>
<li>Virulence / symptoms
<ul>
<li>Though symptoms also come from immune response</li>
</ul></li>
<li>Generation interval; generation time
<ul>
<li>Time-to-infectiousness</li>
</ul></li>
<li>Location of viral replication
<ul>
<li>Colder means closer to the end of the nose</li>
</ul></li>
<li>Mode of transmission</li>
</ul>
<p>What matters particularly at the start? Transmissibility; infectivity; generation time. What doesn’t matter at the start? Immune evasion — because everyone is susceptible. The advantage of immune evasion hardly exists.</p>
<p>Does immune evasion always mean adaptive immune evasion? Because isn’t the first step immunity when there’s animal to human transmission?</p>
<p><span class="math inline">\(R_0\)</span> of wild type was around 3, <span class="math inline">\(R_0\)</span> of alpha was around ~4.5, delta was around ~6 (replicated very quickly)</p>
<p>For delta, a large number of viruses were there before the immune system could catch it — so it was able to infect and transmit from individuals who had immunity, without any of the “rinky-dinky stuff that omicron has.”</p>
<p>What’s selected for later? Immune evasion.</p>
<p>What’s selected for now? We have lots of immunity, lots of types with immune evasion,</p>
<p>Who has heard of “HV.1”? HV.1 is derivative of EG5 which just became the main derivative Mutation in 452, better able to attach to cells. This is similar to what made XBB better than 1Q.1.</p>
<p>Importantly, one cannot have an ever-increasing <span class="math inline">\(R_0\)</span>. Like, an <span class="math inline">\(R_0\)</span> of 8 vs.&nbsp;9 is not as much of a difference between <span class="math inline">\(R_0\)</span> of 2 and 3. In other words, past a certain point, “everyone gets infected,” whereas at the lower boundaries we have differences in the pace of domination.</p>
<p>According to Plague Inc, routes of transmission:</p>
<ul>
<li>Zoonoses:
<ul>
<li>Birds, rodents, domestic animals</li>
</ul></li>
<li>Mosquitos</li>
<li>Blood</li>
<li>Water</li>
<li>Air</li>
<li>Fecal-oral</li>
<li>Aerosol (fomites)</li>
<li>Body Fluids/STDs</li>
<li>Nosocomial</li>
<li>Vector</li>
<li>Foodborne</li>
</ul>
<p>Despite the unrealism of this, there are overlaps and the boundaries of these things are not absolute.</p>
<p><img src="images/aerosol.png" class="img-fluid"></p>
<p>From <a href="https://www.medscape.com/viewarticle/930797" class="uri">https://www.medscape.com/viewarticle/930797</a></p>
<p>Can you think of a disease that has changed its transmission route?</p>
<p>Depends if we’re allowed to think about diseases with changes in its host</p>
<ul>
<li>Avian influenza is fecal-oral changed to respiratory</li>
<li>Mpox</li>
<li>Hepatitis in blood transfusions from STDs</li>
<li>Zika and Mpox</li>
</ul>
<section id="zoonoses" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="zoonoses"><span class="header-section-number">4.1</span> Zoonoses</h2>
<p>Often called a host jump or a host switch (or emergence for a jump into humans).</p>
<p>Distinct from spillover, which is transmission from a reservoir species into another species in which the pathogen does not normally circulate.</p>
<p>A zoonosis is a disease that can be transmitted from animals to humans.</p>
<p><img src="images/zoonosis.png" class="img-fluid"></p>
<p>Anthroponoses is when humans can infect animals.</p>
<p>List zoonotic diseases that are known to jump or have jumped from animals to humans:</p>
<ul>
<li>Avian flu</li>
<li>Ebola</li>
<li>Swine flu</li>
<li>Plague</li>
<li>Malaria</li>
<li>HIV</li>
<li>Yellow fever</li>
<li>Coronavirus</li>
<li>Hantavirus</li>
<li>Measles came from cattle</li>
<li>Bovine encephalitis</li>
<li>Hemoragic fevers</li>
<li>Rabies</li>
<li>Nipah</li>
<li>Chagas</li>
<li>Parasites</li>
</ul>
<p>There is a trade-off refers to the inability to optimize multiple traits simultaneously. Impossible to be well-adapted to</p>
<p>Lots of zoonoses, but not that many cases where it’s become fully transmissible and established in new species. Flu, HIV, COVID, plague.</p>
<p>Adaptation and host switching is not the norm.</p>
<p><img src="images/pathogen_emergence_paper.png" class="img-fluid"></p>
<p>Off-the-shelf: adaptive mutations occur in the animal reservoir, before spillover.</p>
<p>Tailor-made: adaptive mutations occur and are selected following spillover.</p>
<p>We think off-the-shelf is more common.</p>
<p><img src="images/fitness_landscape.png" class="img-fluid"></p>
<p><img src="images/ace2.png" class="img-fluid"></p>
<p>Host jumps can be off-the-shelf (adaptive mutations occur before spillover) or tailor-made (adaptation occurs after spillover).</p>
<p>Off-the-shelf emergence is limited by the probability of the necessary mutations appearing and surviving in the reservoir population.</p>
<p>Fitness landscapes can be very different reservoir and humans representing a tradeoff.</p>
<p>The biggest risk for emergence are generalists because generalists can become specialists.</p>
</section>
<section id="population-size" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="population-size"><span class="header-section-number">4.2</span> Population Size</h2>
<p>Population size is a particularly important and under-appreciated dimension for emergence. How large is the population size in the new host? This determines opportunity for adaptation.</p>
<p><img src="images/popsize.png" class="img-fluid"></p>
<p>Larger populations tend to adapt more quickly. Smaller populations are at risk of losing beneficial adaptations by chance.</p>
<p>Strong (narrow) bottlenecks reduce population size more than a weak (wide) bottlenecks. The founder effect occurs when a new population is “founded” by a small number of individuals whose allele frquencies may differ from the source population.</p>
<p>Types of bottlenecks viruses experiences:</p>
<ul>
<li>Transmission</li>
<li>Loss of host population</li>
<li>Epidemic dynamics (e.g., bottlenecks in the winter)</li>
<li>Selection</li>
</ul>
<p><img src="images/omicron_emergence.png" class="img-fluid"></p>
<p>Petros, Turcinovic et al 2022</p>
<p><a href="https://www.medrxiv.org/content/10.1101/2022.01.27.22269787v1" class="uri">https://www.medrxiv.org/content/10.1101/2022.01.27.22269787v1</a></p>
<p>Effective population size <span class="math inline">\(N_e\)</span>: the population size that would give the same genetic diversity under idealized conditions.</p>
<ul>
<li>Constant population size</li>
<li>Random mating</li>
<li>Neutrality (no selection)</li>
</ul>
<p><span class="math inline">\(N_e\)</span> is the harmonic mean population size</p>
<p><span class="math display">\[N_e = t(\sum_{i=1}^t \frac{1}{N_i})^{-1}\]</span></p>
<p>Skewed by small numbers, meaning <span class="math inline">\(N_e\)</span> is very sensitive to bottlenecks.</p>
<section id="rough-n_es-for-some-famous-things" class="level3" data-number="4.2.1">
<h3 data-number="4.2.1" class="anchored" data-anchor-id="rough-n_es-for-some-famous-things"><span class="header-section-number">4.2.1</span> Rough <span class="math inline">\(N_e\)</span>s for some famous things:</h3>
<ul>
<li><em>E coli</em> ~<span class="math inline">\(10^9\)</span></li>
<li>Influenza 500</li>
<li>Measles 4000</li>
<li>Humans 10,000</li>
</ul>
<p>For influenza, because of the yearly sweeps.</p>
<p>For humans, because of the common ancestors leaving Africa relatively recently.</p>
<ul>
<li>Fraser et al 2009</li>
<li>Bedford et al BMC evol Biol 2011</li>
<li>Takahata Mol Biol Evol 1993</li>
</ul>
</section>
</section>
<section id="bottlenecks-and-host-jumps" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="bottlenecks-and-host-jumps"><span class="header-section-number">4.3</span> Bottlenecks and Host Jumps</h2>
<p>Bottlenecks reduce diversity and slow adaptation.</p>
<p>Strong bottlenecks have a greater effect than weak bottlenecks.</p>
<p>What affects the size of a transmission bottleneck?</p>
<ul>
<li>Abundance in the donor host</li>
<li>Route of transmission</li>
<li>Susceptibility in recipient host</li>
<li>Fitness distribution of the transmitted population</li>
</ul>
</section>
<section id="review" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="review"><span class="header-section-number">4.4</span> Review</h2>
<p>Larger populations adapt more rapidly. Bottlenecks reduce genetic diversity and slow adaptation. The effective population size <span class="math inline">\(N_e\)</span> is the size of an idealized population that would have the same amount of genetic diversity.</p>
<p>Bottlenecks may substantially reduce <span class="math inline">\(N_e\)</span>.</p>
<p>Remember that most zoonoses are weakly transmissible in humans.</p>
<p><img src="images/r0_1.png" class="img-fluid"> <img src="images/r0_2.png" class="img-fluid"></p>
<p><em>Evolutionary rescue</em> is improving the reproductive rate before going extinct.</p>
<p><img src="images/evolutionary_rescue.png" class="img-fluid"></p>
<p>Example: Original SARS may have almost been rescued due to a deletion of ORF8.</p>
<p><img src="images/drug_resistance.png" class="img-fluid"></p>
<p>One reason is that there already exist large diversity in bacteria.</p>
</section>
<section id="one-dose-or-two" class="level2" data-number="4.5">
<h2 data-number="4.5" class="anchored" data-anchor-id="one-dose-or-two"><span class="header-section-number">4.5</span> One Dose or Two</h2>
<p><img src="images/one_dose_or_two.png" class="img-fluid"> <img src="images/one_dose_or_two2.png" class="img-fluid"></p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../week1/week1.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Week 1</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
  </div>
</nav>
</div> <!-- /content -->



</body></html>