<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.269">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>EPI519 Evolutionary Epidemiology - 4&nbsp; Week 4</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../week3/week3.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


</head>

<body class="nav-sidebar floating">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
    <div class="container-fluid d-flex justify-content-between">
      <h1 class="quarto-secondary-nav-title"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Week 4</span></h1>
      <button type="button" class="quarto-btn-toggle btn" aria-label="Show secondary navigation">
        <i class="bi bi-chevron-right"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="../">EPI519 Evolutionary Epidemiology</a> 
    </div>
      </div>
      <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
      </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../index.html" class="sidebar-item-text sidebar-link">Preface</a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../week1/week1.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Week 1</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../week2/week2.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Week 2</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../week3/week3.html" class="sidebar-item-text sidebar-link"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Week 3</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../week4/week4.html" class="sidebar-item-text sidebar-link active"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Week 4</span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#guest-lecture-from-duncan-maccannell-pathogen-genomics-in-public-health" id="toc-guest-lecture-from-duncan-maccannell-pathogen-genomics-in-public-health" class="nav-link active" data-scroll-target="#guest-lecture-from-duncan-maccannell-pathogen-genomics-in-public-health"><span class="toc-section-number">5</span>  Guest Lecture from Duncan MacCannell, Pathogen Genomics in Public Health</a></li>
  <li><a href="#advanced-molecular-detection-amd" id="toc-advanced-molecular-detection-amd" class="nav-link" data-scroll-target="#advanced-molecular-detection-amd"><span class="toc-section-number">6</span>  Advanced Molecular Detection (AMD)</a>
  <ul class="collapse">
  <li><a href="#areas-of-emphasis" id="toc-areas-of-emphasis" class="nav-link" data-scroll-target="#areas-of-emphasis"><span class="toc-section-number">6.1</span>  Areas of Emphasis</a></li>
  <li><a href="#workflow" id="toc-workflow" class="nav-link" data-scroll-target="#workflow"><span class="toc-section-number">6.2</span>  Workflow</a></li>
  <li><a href="#foodborne-disease-surveillance" id="toc-foodborne-disease-surveillance" class="nav-link" data-scroll-target="#foodborne-disease-surveillance"><span class="toc-section-number">6.3</span>  Foodborne Disease Surveillance</a></li>
  <li><a href="#growing-phl-ngs-capacity" id="toc-growing-phl-ngs-capacity" class="nav-link" data-scroll-target="#growing-phl-ngs-capacity"><span class="toc-section-number">6.4</span>  Growing PHL NGS Capacity</a></li>
  <li><a href="#influenza-vaccine-strain-selection" id="toc-influenza-vaccine-strain-selection" class="nav-link" data-scroll-target="#influenza-vaccine-strain-selection"><span class="toc-section-number">6.5</span>  Influenza Vaccine Strain Selection</a></li>
  <li><a href="#molecular-epidemiology-of-cyclosporiasis" id="toc-molecular-epidemiology-of-cyclosporiasis" class="nav-link" data-scroll-target="#molecular-epidemiology-of-cyclosporiasis"><span class="toc-section-number">6.6</span>  Molecular Epidemiology of Cyclosporiasis</a></li>
  <li><a href="#us-sars-cov-2-sequencing-december-31-2020" id="toc-us-sars-cov-2-sequencing-december-31-2020" class="nav-link" data-scroll-target="#us-sars-cov-2-sequencing-december-31-2020"><span class="toc-section-number">6.7</span>  US SARS-CoV-2 Sequencing December 31, 2020</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title d-none d-lg-block"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">Week 4</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="guest-lecture-from-duncan-maccannell-pathogen-genomics-in-public-health" class="level1" data-number="5">
<h1 data-number="5"><span class="header-section-number">5</span> Guest Lecture from Duncan MacCannell, Pathogen Genomics in Public Health</h1>
<p>Director of Office of Advanced Molecular Detection (AMD)</p>
<p>Division of Infectious Disease Readiness and Innovation</p>
<p>National Center for Emerging and Zoonotic Infectious Diseases at CDC</p>
<p><span class="citation" data-cites="drm">@drm</span><span class="citation" data-cites="mstdn.science">@mstdn.science</span></p>
<p>cdc.gov/amd</p>
</section>
<section id="advanced-molecular-detection-amd" class="level1" data-number="6">
<h1 data-number="6"><span class="header-section-number">6</span> Advanced Molecular Detection (AMD)</h1>
<p>AMD started at end of 2013 (FY2014) as a $30m/yr, now $40m/yr budget.</p>
<p>Five objectives:</p>
<ol type="1">
<li>Improved pathogen detection &amp; characterization</li>
<li>Enable new diagnostic methods to meet public health needs</li>
<li>Build genomic &amp; bioinformatics capabilities</li>
<li>Implement enhanced, sustainable, integrated information systems</li>
<li>Develop tools for prediction, modeling, and early recognition of emerging infectious threats.</li>
</ol>
<p>New CDC center for forecasting infectious diseases, new centers for excellence initiative provides promise on the predictive front.</p>
<section id="areas-of-emphasis" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="areas-of-emphasis"><span class="header-section-number">6.1</span> Areas of Emphasis</h2>
<p>Advanced laboratory technologies/approaches:</p>
<ul>
<li>NGS for pathogen (meta)genomics, (meta)transcriptonomics</li>
<li>Mass spectrometry, CRISPR</li>
<li>Platforms: Illumina, PacBio, IonTorrent, Oxford Nanopore, etc.</li>
<li>Other tech: MALDI-ToF, CRISPR, proteomics, microfluidics, etc.</li>
<li>CDC Facilities; state/local public health laboratories, field.</li>
</ul>
<p>IT:</p>
<ul>
<li>IT infra; storage; networking, high-performance computing, cloud</li>
<li>Bioinformatics and scientific computing core support</li>
<li>CDC High Performance Computing Center of Excellence</li>
<li>Lab/epi data aggregation, integration, visualization and reporting</li>
</ul>
<p>Integration remains a problem: most data stop at the laboratory door.</p>
<p>Strategic coordination and program support</p>
<ul>
<li>Funding 30+ CDC AMD project activities</li>
<li>Promotion of standardized platforms, methods, shared resources, bioinformatics, and QMS</li>
<li>Coordination of state-based activities &amp; initiatives</li>
<li>Connections with federal (FDA, NIH, NIST, DoD, DHS)</li>
</ul>
<p>Workforce development</p>
<ul>
<li>Training, recruitment, administrative, fellowships</li>
</ul>
</section>
<section id="workflow" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="workflow"><span class="header-section-number">6.2</span> Workflow</h2>
<p>Input: DNA/RNA</p>
<p>Source: Genomic, Amplicon, Whole Sample</p>
<p>Host/vector/pathogen/environment</p>
<p>=&gt;</p>
<p>NGS Workflow:</p>
<p>Platforms chemistry, performance characteristics, labor/TaT Expertise Cost</p>
<p>=&gt;</p>
<p>Bioinformatics Workflow:</p>
<ul>
<li>Hardware/software</li>
<li>Specialized skillsets</li>
<li>Algorithmics/pipelines</li>
<li>Pathogen databases</li>
<li>Data analysis/interpret/integration/visualization</li>
</ul>
<p>=&gt;</p>
<p>Output: Information from Sequence Data</p>
<ul>
<li>Comparative genomics
<ul>
<li>Identification</li>
<li>High res Straintyping/Subtyping</li>
<li>Cluster identification</li>
<li>Molecular evolution</li>
<li>Genotypic characterization</li>
<li>Virulence, AR, Signatures</li>
<li>Functional annotation</li>
<li>Diagnostic dev/validation</li>
<li>Minor populations, quasispecies</li>
<li>Host/pathogen expression</li>
</ul></li>
<li>Metagenomics
<ul>
<li>Pathogen identification/discovery</li>
<li>Culture-independent diagnostics</li>
<li>Microbial ecology/diversity</li>
</ul></li>
</ul>
<p>For example, we might be looking for markers of bioengineering for biosecurity monitoring.</p>
<p>It’s hard to stress just how important new bioinformatics workflows have been in affecting public health. Even just a few years ago pulse gel electrophoresis and fragment sequencing were still quite niche and often had to be pathogen-specific.</p>
<p>Primary Databases: SRA, GenBank, open public databases. Quality of annotation varies, quality varies. One really needs a level of expert curation to create a high quality database. It’s grunt-work that needs to be done.</p>
<p><a href="https://www.dhs.gov/biowatch-program">BioWatch program</a> screens for known pathogens in the air. The problem is what constitutes a signal?</p>
<p>Metagenomics is still only almost cost-effective. The computational challenges are still quite high.</p>
</section>
<section id="foodborne-disease-surveillance" class="level2" data-number="6.3">
<h2 data-number="6.3" class="anchored" data-anchor-id="foodborne-disease-surveillance"><span class="header-section-number">6.3</span> Foodborne Disease Surveillance</h2>
<p>~48 million foodborne illnesses occur in the US each year, resulting in 128,000 hospital admissions and 3,000 deaths.</p>
<p>Although most cases occur as part of unrecognized outbreaks, &gt;1,000 outbreaks are reported each year.</p>
<p>PulseNet, a national network of public health and food regulatory agency laboratories, has played a key role in identifying and stopping outbreaks of foodborne illness.</p>
<p>Pathogens/foods linked to most foodborne illnesses:</p>
<ul>
<li>Campylobacter (poultry)</li>
<li>…</li>
</ul>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/pulsenet.jpg" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">PulseNet</figcaption><p></p>
</figure>
</div>
<p><a href="https://www.cdc.gov/foodsafety/foodborne-germs.html" class="uri">https://www.cdc.gov/foodsafety/foodborne-germs.html</a></p>
<p>WGS-based analysis of putative multistate of PFGE-indistinguishable Salmonella Heidelberg (Xbal: JF6X01.0022)</p>
<p>Contemporaneous timeframe, similar presentations, possible link?</p>
</section>
<section id="growing-phl-ngs-capacity" class="level2" data-number="6.4">
<h2 data-number="6.4" class="anchored" data-anchor-id="growing-phl-ngs-capacity"><span class="header-section-number">6.4</span> Growing PHL NGS Capacity</h2>
<p>Growing applications:</p>
<ul>
<li>PulseNet, influenza, Healthcare-associated infections, bacterial meningitis, Cryptococcus, Cyclospora, Viral hepatitis, Gonococcus, Influenza, Legionella, Malaria, Streptococcus, Tickborne diseases, Tuberculosis,…</li>
</ul>
<p><a href="https://pubmed.ncbi.nlm.nih.gov/29047425/" class="uri">https://pubmed.ncbi.nlm.nih.gov/29047425/</a></p>
<p><a href="https://wwwnc.cdc.gov/eid/article/23/11/17-0308_article" class="uri">https://wwwnc.cdc.gov/eid/article/23/11/17-0308_article</a></p>
<p>Satellite imagery helped locate cooling towers.</p>
<p>Outbreak of Extensively Drug-resistant Pseudomonas aeruginosa Associated with Artificial Tears <a href="https://www.cdc.gov/hai/outbreaks/crpa-artificial-tears.html" class="uri">https://www.cdc.gov/hai/outbreaks/crpa-artificial-tears.html</a></p>
<p>Caused multiple cases of blindness, several cases of death.</p>
</section>
<section id="influenza-vaccine-strain-selection" class="level2" data-number="6.5">
<h2 data-number="6.5" class="anchored" data-anchor-id="influenza-vaccine-strain-selection"><span class="header-section-number">6.5</span> Influenza Vaccine Strain Selection</h2>
<p>Wentworth/Barnes/Neuhaus (CDC/NCIRD/ID)</p>
<p>The old way was:</p>
<p>Specimen collection -&gt; Isolation and propagation -&gt; Phenotypic analysis (in a host species) -&gt; Traditional Sanger Sequencing</p>
<p>New model:</p>
<p>Specimen collection -&gt; 10,000 per year Sequenced in ~1 week -&gt; Isolation and Propagation -&gt; Phenotypic Analysis</p>
<p>Facilitated by next generation sequencing (MiSeq and NextSeq).</p>
<p>Rambo-Martin, Keller, et al.&nbsp;bioRxiv 2019 https://doi.org/10.1101/585588 <a href="https://www.biorxiv.org/content/10.1101/585588v1" class="uri">https://www.biorxiv.org/content/10.1101/585588v1</a></p>
<p>Hepatitis C Virus Outbreak —&nbsp;HCW</p>
<p>18 cases, index case was HCW</p>
<p>154,233 error-corrected reads</p>
<p>33,767 unique sequences</p>
<p>Each node is a unique sequence</p>
<p>Different patients are shown in different colors</p>
<p>Two sequences are linked if they differ at a single nucleotide position</p>
<p>Network analysis using GHOST</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/hepc_ghost.jpg" class="img-fluid figure-img"></p>
<p></p><figcaption class="figure-caption">Network of Hepatitis C from GHOST</figcaption><p></p>
</figure>
</div>
<p>Multi-state outbreak of Meningitis from a New England lab not operating up to code <a href="https://www.cdc.gov/hai/outbreaks/meningitis-facilities-map.html" class="uri">https://www.cdc.gov/hai/outbreaks/meningitis-facilities-map.html</a></p>
<p>Litvintseva et al (JCM 2014)</p>
<p>Pathogen identified as Fungal Meningitis, Exserohilum rostratum, not typically a human pathogen.</p>
<p>Fungal Meningitis Outbreak Associated with Procedures Performed under Epidural Anesthesia in Matamoros, Mexico <a href="https://www.cdc.gov/hai/outbreaks/meningitis-epidural-anesthesia.html" class="uri">https://www.cdc.gov/hai/outbreaks/meningitis-epidural-anesthesia.html</a></p>
<p>Multistate Outbreak of Melioidosis Associated with Imported Aromatherapy Spray, NEJM, 2022 <a href="https://www.nejm.org/doi/full/10.1056/NEJMoa2116130" class="uri">https://www.nejm.org/doi/full/10.1056/NEJMoa2116130</a></p>
<p><img src="images/nejm_spray.jpeg" class="img-fluid"></p>
</section>
<section id="molecular-epidemiology-of-cyclosporiasis" class="level2" data-number="6.6">
<h2 data-number="6.6" class="anchored" data-anchor-id="molecular-epidemiology-of-cyclosporiasis"><span class="header-section-number">6.6</span> Molecular Epidemiology of Cyclosporiasis</h2>
<p>Ten years ago, molecular typing of <em>Cyclospora cayetanensis</em> was all but impossible.</p>
<p>Advances in isolation methods, sequencing, and bioinformatics has enabled reliable typing of the parasite.</p>
<p>After applying these methods and building a database including thousands of reference sequences, CDC scientists discovered that there are not one but three highly diverged populations of Cyclospora that will not interbreed.</p>
<p><a href="https://www.cdc.gov/amd/whats-new/cyclospora.html" class="uri">https://www.cdc.gov/amd/whats-new/cyclospora.html</a></p>
<p>This finding has significant implications for surveillance, prevention and source attribution.</p>
<p>This is a new era in genomic parasitology.</p>
<p>Joel Barratt, NCEZID DPDM*</p>
<p>cdc.gov/amd/whats-new/cyclospora.html</p>
<p>Locally acquired cases of malaria (2023): rapidly developed an AmpliSeq-based method focused on <em>P. vivax</em> and comprising more complex marker panels than MaRS combined with the cyclone analytical approach.</p>
<p>Brian Raphael, Molly Freeman, Joel Barrett, DPDM</p>
</section>
<section id="us-sars-cov-2-sequencing-december-31-2020" class="level2" data-number="6.7">
<h2 data-number="6.7" class="anchored" data-anchor-id="us-sars-cov-2-sequencing-december-31-2020"><span class="header-section-number">6.7</span> US SARS-CoV-2 Sequencing December 31, 2020</h2>
<p>19.8M US Cases, 57.3k sequences, including 20 SPHLs.</p>
<p>The SPHERES consortium: https://www.cdc.gov/coronavirus/2019-ncov/variants/spheres.html</p>
<p><img src="images/spheres.jpg" class="img-fluid"></p>
<p>Where are we now?</p>
<p>At 16 million global sequences, SARS-CoV-2 is the most extensively sequenced pathogen in history.</p>
<p>Of those 16 million global sequences, about 1/3 are from the US.</p>
<p>21% are from local/state/federal public health surveillance</p>
<p>Trying to do phylogenetic analysis on the scale of ~6.5-million sequences, the bayesian approaches we typically go to totally break down.</p>
<p>Other considerations:</p>
<ul>
<li>New Wastewater systems</li>
<li>One Health</li>
<li>New <a href="https://wwwnc.cdc.gov/travel">traveler’s health</a> system</li>
</ul>
<p>Barriers to genomic surveillance in public health practice:</p>
<ul>
<li>Sample Acquisition
<ul>
<li>Selection &amp; Prioritization</li>
<li>Maintaining cold-chain</li>
<li>Representativeness</li>
</ul></li>
<li>Laboratory
<ul>
<li>Deficiencies in q-tips</li>
<li>Need to roboticize</li>
<li>Supply chain</li>
<li>Instruments</li>
<li>Vendor lock-in</li>
</ul></li>
<li>Workforce
<ul>
<li>How do you maintain an effective workforce?</li>
</ul></li>
<li>Information Technology
<ul>
<li>How do you provide these technology when most state health departments already have vendor contracts?
<ul>
<li>This is a major reason why so much sequencing is submitted ot be done done at CDC</li>
</ul></li>
</ul></li>
<li>Quality Management
<ul>
<li>Example datasets</li>
</ul></li>
<li>Bioinformatics and Genomic Epidemiology
<ul>
<li>Global skills gap</li>
<li>Complex workflows</li>
<li>Benefits/deficiencies of open source</li>
<li>i18n</li>
</ul></li>
<li>Communication/Dissemination
<ul>
<li>Dashboarding / data exploration / dataviz</li>
<li>Collaboration / partnership</li>
</ul></li>
<li>Reporting/interpretation and data sharing</li>
</ul>
<p>There is a new CDC/APHL fellowship program —&nbsp;everything from software engineering.</p>
<p><a href="https://www.cdc.gov/amd/training/covid-19-gen-epi-toolkit.html" class="uri">https://www.cdc.gov/amd/training/covid-19-gen-epi-toolkit.html</a></p>
<p><a href="https://covariants.org/" class="uri">https://covariants.org/</a> <a href="https://outbreak.info/" class="uri">https://outbreak.info/</a> <a href="https://scholar.harvard.edu/cvt/home" class="uri">https://scholar.harvard.edu/cvt/home</a></p>
<p>Pathogen Genomics Centers of Excellence</p>
<p><a href="https://www.cdc.gov/amd/resources/scientific-superhero-posters.html" class="uri">https://www.cdc.gov/amd/resources/scientific-superhero-posters.html</a></p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../week3/week3.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Week 3</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
  </div>
</nav>
</div> <!-- /content -->



</body></html>